spec/models/news_item_spec.rb

```diff
@@ -1,5 +1,63 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe NewsItem, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "validation" do
+    it { should validate_presence_of(:title) }
+    it { should validate_length_of(:title).
+                  is_at_most(ActiveRecordExtensions::MAX_STRING_COLUMN_LENGTH) }
+    it { should validate_presence_of(:body) }
+    it { should validate_length_of(:body).
+                  is_at_most(ActiveRecordExtensions::MAX_TEXT_COLUMN_LENGTH) }
```

Comment:
Expression at 10, 78 should be on its own line.<br>Use 2 (not 14) spaces for indenting an expression spanning multiple lines.